Thales/thales_safenet/_meta/fields.yml

```
@@ -0,0 +1,38 @@
+safenet.sourceid:
+  name: safenet.sourceid
+  type: keyword
+  description: "original event id"
+
+safenet.tenantid:
+  name: safenet.tenantid
+  type: keyword
+  description: "id of event tenant"
+
+safenet.globalaccessid:
+  name: safenet.globalaccessid
+  type: keyword
+  description: "global Access ID"
```

```suggestion
  description: "Global Access ID"
```

Is Global Access like a product name ?